README.md

```diff
@@ -0,0 +1,52 @@
+The OpenXT synchronizers
+
+![Block diagram of the OpenXT Synchonizer](diagrams/Syncstruct.png)
+
+Synchronizer XT enables sysadmins and IT professionals to securely and
+easily manage a large number of XenClient laptops and the Windows VMs
+that run on them.
+
+This contains the client part of the OpenXT synchronizer. The code here (mainly 
+sync_client/client.py) is a python program that runs in a low privilege service VM,
+makes an HTTPS to a synchronizer web server which tells it what VMs it should be running.
+client.py then compares this to the current state of the machine, downloads disks, configures
+and starts VM as appropriate.
+
+Note that client.py is a short running process which maintains no state. It runs once
+to completion and then exits. Normally client.py is started by launcher.py which handles
+getting the configuring information for client.py and running it regularly, and handling exits.
+
+## Dependencies
+
+sync-client requires:
+
+* a synchronizer server to get target state and VHDs from (see https://github.com/openxt/sync-server/). In
+turn the synchronizer server will require:
```

Comment:
Probably shouldn't cite what synchronizer requires on the server side. That should be in sync-server which is already referenced here